doc/psud/PSU_daemon_design.md

```diff
@@ -95,14 +166,41 @@ Commands:
 
 The current output for "show platform psustatus" looks like:
 
+```
+admin@sonic:~$ show platform  psustatus
+PSU    Model          Serial        HW Rev      Voltage (V)    Current (A)    Power (W)  Status  LED
+-----  -------------  ------------  --------  -------------  -------------  -----------  ------- -----
+PSU 1  MTEF-PSF-AC-A  MT1629X14911  A3                12.08           5.19        62.62  WARNING green
```

Comment:
The WARNING status here could be more clear.  Indicate the condition is due to the power budget and not something else (e.g. overtemp of PSU in the future).  Maybe something like "PWR EXCEEDED" or something similar.

Reply:
Will fix it.